pkg/cloud/amazon/s3_storage.go

```diff
@@ -496,7 +497,36 @@ func newLogAdapter(ctx context.Context) *awsLogAdapter {
 	}
 }
 
-var awsVerboseLogging = aws.LogLevel(aws.LogDebugWithRequestRetries | aws.LogDebugWithRequestErrors)
+// TODO(during review): Do we want others?
+// LogSigning ClientLogMode = 1 << (64 - 1 - iota)
+// LogRetries
+// LogRequest
+// LogRequestWithBody
+// LogResponse
+// LogResponseWithBody
+// LogDeprecatedUsage
+// LogRequestEventMessage
+// LogResponseEventMessage
```

Comment:
The documentation is rather sparse on the exact definition of some of these.

I could see having two debug levels. One that is what you have below, and another that includes LogRequest | LogResponse.  I don't think we'll ever want to log bodies.